context/app/static/js/helpers/trackers.js

```diff
@@ -85,7 +85,8 @@ const makeFaroSafe = (obj) =>
   Object.fromEntries(Object.entries(obj).map(([key, value]) => [key, `${JSON.stringify(value)}`]));
 
 function trackEvent(event) {
-  tracker.trackEvent(event);
+  const name = event?.label ?? {};
```

Comment:
Should the fallback here be `''` instead of `{}`? Otherwise I think it would show up in matomo as `[Object Object]`